src/BrowsershotLambda.php

```diff
@@ -54,4 +56,32 @@ protected function throwError(SettledResult $response): void
 
         $response->throw();
     }
+
+    /**
+     * @throws CouldNotTakeBrowsershot
+     */
+    public function saveToS3(string $targetPath, string $disk = 's3')
+    {
+        $this->setOption('s3', [
+            'path' => $targetPath,
+            'region' => config('sidecar.region'),
+            'bucket' => config("filesystems.disks.$disk.bucket")
+        ]);
```

Comment:
I suggest using the same region the disk uses in case the lambda is in a different region. In addition, here are the suggested `key` and `secret` credential options.

```suggestion
        $this->setOption('s3', [
            'path' => $targetPath,
            'region' => config("filesystems.disks.{$disk}.region"),
            'bucket' => config("filesystems.disks.{$disk}.bucket"),
            'key' => config("filesystems.disks.{$disk}.key") ?? $_ENV['AWS_ACCESS_KEY_ID'] ?? null,
            'secret' => config("filesystems.disks.{$disk}.secret") ?? $_ENV['AWS_SECRET_ACCESS_KEY'] ?? null,
        ]);
```